yo/app/views/metadata.html

```diff
@@ -0,0 +1,77 @@
+<div ng-model="tabs" id="board" ng-controller="MetadataController">
```

Comment:
This code appears almost word for word in datasetsBrowser.html. Are they duplicate? Do you need both?